docs/SUPPORT.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Support
+
+  * To report any issue, create an issue [here](https://github.com/dell/<repo name>/issues).
+  * If any requirements have not been addressed, then create an issue [here](https://github.com/dell/<repo name>/issues).
+  * To provide feedback to the development team, send an email to **<email_id>**.
```

Comment:
I don't think we should use e-mail to provide/receive feedback. If folks have feedback they should just open an issue. The development team is subscribed to the repositories so we will get notified via e-mail anyway.